charts/connectors/values.yaml

```diff
@@ -0,0 +1,247 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains values for variables referenced from yaml files in the templates directory.
+#
+# For further information on Helm templating see the documentation at:
+#  https://helm.sh/docs/chart_template_guide/values_files/
+
+#
+# >>> This chart requires Helm version 3.6.0 or greater <<<
+#
+
+# Common settings
+#
+# -- Override `connectors.name` template.
+nameOverride: ""
+# -- Override `connectors.fullname` template.
+fullnameOverride: ""
+# -- Additional labels to add to all Kubernetes objects.
+# For example, `my.k8s.service: redpanda`.
+commonLabels: {}
+# -- Taints to be tolerated by Pods, can override this for StatefulSets.
+# For details,
+# see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).
+tolerations: []
+
+# -- Redpanda Docker image settings.
+image:
+  # -- Docker repository from which to pull the Redpanda Docker image.
+  repository: docker.redpanda.com/redpandadata/connectors
+  # -- The Redpanda version.
+  # See DockerHub for:
+  # [All stable versions](https://hub.docker.com/r/redpandadata/redpanda/tags)
+  # and [all unstable versions](https://hub.docker.com/r/redpandadata/redpanda-unstable/tags).
+  # @default -- `Chart.appVersion`.
+  tag: ""
+  # -- The imagePullPolicy.
+  # If `image.tag` is 'latest', the default is `Always`.
+  pullPolicy: IfNotPresent
+
+# -- Pull secrets may be used to provide credentials to image repositories
+# See https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
+
+connectors:
+  # -- Bootstrap servers list is coma separated list of individual Redpanda brokers as single line string
+  bootstrapServers: ""
+  # -- List of Redpanda IP:Port or DNS:Port separated by coma.
+  schemaRegistryURL: ""
+  # -- TODO the schema
+  additionalConfiguration: ""
+  secretManager:
+    enabled: false
+    region: ""
+    consolePrefix: ""
+    connectorsPrefix: ""
+  producerBatchSize: 131072
+  producerLingerMS: 1
+  storage:
+    replicationFactor:
+      offset: -1
+      config: -1
+      status: -1
+    remote:
+      read:
+        offset: false
+        config: false
+        status: false
+      write:
+        offset: false
+        config: false
+        status: false
+    topic:
+      offset: _internal_connectors_offsets
+      config: _internal_connectors_configs
+      status: _internal_connectors_status
+  groupID: connectors-cluster
+
+# -- Authentication settings.
+# For details,
+# see the [SASL documentation](https://docs.redpanda.com/docs/manage/kubernetes/security/sasl-kubernetes/).
+auth:
+  sasl:
+    # -- The authentication mechanism to use for the superuser. Options are `scram-sha-256` and `scram-sha-512`.
+    mechanism: scram-sha-256
```

Comment:
we default to 512 in the redpanda chart, should we do the same here?

Reply:
sure I will switch to 512